euth_wagtail/templates/account/howto_item.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n static thumbnail %}
+
+<div class="howto-item__container">
+  <h1>{{ howto_item.header }}</h1>
+  <img class="howto-item__image" alt="how to register image" src="{% static howto_item.img %}">
```

Comment:
for a11y reasons, there should only be a single h1 element on each page, in this case I think it should be 'Registration' or 'Login' the slide headings can be h2 or h3, i'm never quite sure in such a mixed use page, whatever looks better is sometimes the best option